### PR TITLE
fix broken au/tasmania source

### PR DESCRIPTION
**Pull request by andrewharvey**
...however this data doesn't appear to be under an open license https://www.tas.gov.au/stds/codi.htm, so not sure of the history here.

I'll email the data provider, but leading into the holidays probably won't be 2 weeks before I hear back.

### CODE REVIEW

File: adapters/tasmania.js [hunk 1]
```diff
@@ -36,63 +36,23 @@ var formatData = function (data, source) {
     var date = moment.tz(string, 'HHmmss', 'Australia/Hobart');
     return {utc: date.toDate(), local: date.format()};
   };
-  // manually retrieved list of station names
```
Comment (andrewharvey) [inline adapters/tasmania.js]: this was removed since the station name is now provided in the dataset

File: adapters/tasmania.js [hunk 2]
```diff
@@ -36,63 +36,23 @@ var formatData = function (data, source) {
...
-    'CY': 'Cygnet',
-    'GV': 'Geeveston'
-  };
 
   var output = [];
   var measurements = [];
 
   // parse the csv feed, exclude # lines
-  output = parse(data, {trim: true, comment: '#'});
+  output = parse(data, {trim: true, comment: ' #'});
```
Comment (andrewharvey) [inline adapters/tasmania.js]: this fixes the source data parsing, and fixes the broken adaptor

**Comment by andrewharvey** [comment]
I sent a follow up email yesterday about the licensing.

**Comment by andrewharvey** [comment]
> I sent a follow up email yesterday about the licensing.

Sent another follow up email about licensing.

**Comment by majesticio** [comment]
I am closing this PR as we have an adapter in place. As always we appreciate your contributions and your time.

**Pull request closed without merging**